corehq/apps/api/tests/core_api.py

```diff
@@ -398,7 +398,7 @@ def setUp(self):
                                                  None, None)
 
     def tearDown(self):
-        self.commcare_user.delete(self.domain, deleted_by=None)
+        self.commcare_user.delete(self.domain.name, deleted_by=None)
```

Comment:
I guess the `domain_obj` convention hasn't made it all the way throughout HQ.

Reply:
There were few of these instances, mostly in tests iirc